config/locales/en/application.yml

```diff
@@ -7,3 +7,5 @@ en:
       support_and_feedback: Support and feedback
       documentation: Documentation
       how_to_write_publish_improve: How to write, publish, and improve content
+    back_link:
+      back: Back to document list
```

Comment:
This looks to be a global change - and we only want to change this for a specific view.

Reply:
I wasn't sure whether to hardcode this in that view or put it in a locale file and thought it might go here. Should I just hardcode it in the view?

Reply:
Yeah I'd hardcode it as that seems consistent with: https://github.com/alphagov/content-publisher/blame/6ec7337fea74534cce1ea515633a72981cd936c3/docs/adr/0004-editing-microcopy.md#L43